web/client/components/details/DetailsPanel.jsx

```diff
@@ -56,50 +64,67 @@ class DetailsPanel extends React.Component {
+                        glyph: 'arrow-left',
+                        tooltipId: 'details.cancel',
+                        onClick: () => this.props.onCancelEdit()
+                    }, {
+                        disabled: this.props.loadFlags.detailsSaving,
+                        glyph: 'floppy-disk',
+                        tooltipId: 'details.save',
+                        onClick: () => this.props.onSave()
+                    }] : [{
+                        glyph: 'pencil',
+                        tooltipId: 'details.edit',
+                        onClick: () => this.props.onEdit()
+                    }, {
+                        glyph: 'cog',
+                        tooltipId: 'details.settings',
+                        onClick: () => this.props.onEditSettings()
+                    }])]}/>
+            </div>
+        );
 
         return (
             <ResizeDetector handleWidth>
                 { ({ width }) =>
                     <div className="react-dock-no-resize">
-                        <Dock dockStyle={this.props.dockStyle} {...this.props.dockProps} isVisible={this.props.active} fluid size={this.props.width / width > 1 ? 1 : this.props.width / width}>
-                            <Panel id={this.props.id} header={panelHeader} style={this.props.panelStyle} className={this.props.panelClassName}>
-                                <BorderLayout>
-                                    <div className="ms-details-preview-container">
-                                        {!this.props.detailsText ?
-
-                                            <Spinner spinnerName="circle" noFadeIn overrideSpinnerClassName="spinner"/> :
-                                            <div className="ms-details-preview" dangerouslySetInnerHTML={{ __html:
-                                                this.props.detailsText === NO_DETAILS_AVAILABLE
-                                                    ? LocaleUtils.getMessageById(this.context.messages, "maps.feedback.noDetailsAvailable")
-                                                    : this.props.detailsText
-
-                                            }} />}
+                        <DockPanel
+                            fluid
+                            open={this.props.show}
+                            size={this.props.width / width > 1 ? 1 : this.props.width / width}
+                            position="right"
+                            bsStyle="primary"
+                            title={<Message msgId="details.title"/>}
+                            onClose={() => this.props.onClose()}
+                            glyph="sheet"
+                            style={this.props.dockStyle}>
+                            <Panel id={this.props.id} style={this.props.panelStyle} className={this.props.panelClassName}>
+                                <BorderLayout
+                                    header={this.props.canEdit ? header : undefined}>
```

Comment:
This is different on how the modal and the panel shows the details content and toolbars.

I would like to unify them where the container change but the content is the same

![image](https://user-images.githubusercontent.com/11991428/82672264-6d12a580-9c40-11ea-95b9-5c11d230ef16.png)

vs 
![image](https://user-images.githubusercontent.com/11991428/82672302-7865d100-9c40-11ea-9a60-b150ffb7ec3f.png)